doc/api/readline.markdown

```diff
@@ -39,6 +39,8 @@ the following values:
    treated like a TTY, and have ANSI/VT100 escape codes written to it.
    Defaults to checking `isTTY` on the `output` stream upon instantiation.
 
+ - `historySize` - maximum number of history lines retained. Defaults to `30`.
```

Comment:
Leaving this option at 30 so as not to modify existing userland repls. The internal CLI repl now defaults to 1000, though.